### PR TITLE
Zakmes

### PR DESCRIPTION
**Pull request by ruuttt**
@casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2786)

### CODE REVIEW

File: dutch/over-het-hek/zakmes.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Het zakmes bevat een ijzerzaag. Die is nog flijmscherp want je hebt hem nog nooit gebruikt.
```
Comment (casey-collab) [inline dutch/over-het-hek/zakmes.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline dutch/over-het-hek/zakmes.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ruuttt** [comment]
moved zakmes to subfolder

**Comment by ruuttt** [comment]
@casey-collab

File: dutch/over-het-hek/zakmes/zakmes.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Het zakmes bevat een ijzerzaag. Die is nog flijmscherp want je hebt hem nog nooit gebruikt.
```
Comment (casey-collab) [inline dutch/over-het-hek/zakmes/zakmes.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ruuttt** [comment]
added end of line to hek.md @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**